tools/gulptasks/dashboards/scripts-watch.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) Highsoft AS
+ */
+
+const gulp = require('gulp');
+
+/* *
+ *
+ *  Constants
+ *
+ * */
+
+const WATCH_GLOBS = [
+    'ts/Dashboards/**/*.ts',
+    'ts/Data/**/*.ts',
+    'ts/DataGrid/**/*.ts',
+    'ts/Shared/**/*.ts',
+    'ts/masters-dashboards/**/*.ts',
+    'css/**/*.css'
+];
+
+/* *
+ *
+ *  Tasks
+ *
+ * */
+
+/**
+ * Continuesly watching sources to restart the `scripts-js` task.
+ *
+ * @return {Promise<void>}
+ *         Promise to keep
+ */
+async function task() {
+
+    const argv = require('yargs').argv;
+    const fsLib = require('../lib/fs');
+    const logLib = require('../lib/log');
+    const processLib = require('../lib/process');
+
+    if (processLib.isRunning('dashboards/scripts-watch')) {
+        logLib.warn('Running watch process detected.');
+        if (!argv.force) {
+            logLib.warn('Skipping task...');
+            return Promise.resolve();
+        }
+    }
+
+    let jsHash;
+    let cssHash;
+
+    gulp
+        .watch(WATCH_GLOBS, { queue: true }, done => {
+
+            const buildTasks = [];
+            const newJsHash = fsLib.getDirectoryHash('js', true);
+
+            if (newJsHash !== jsHash || argv.force || argv.dts) {
+                jsHash = newJsHash;
+                buildTasks.push('dashboards/scripts');
+                if (argv.dts) {
+                    buildTasks.task('jsdoc-dts')();
+                }
```

Comment:
I meant the whole `if` statement can be deleted. We do not need to run `jsdoc-dts` for Dashboards.

Reply:
Fixed.